colorbleed/plugins/fusion/publish/validate_background_depth.py

```diff
@@ -0,0 +1,41 @@
+import pyblish.api
+
+from colorbleed import action
+
+
+class ValidateBackgroundDepth(pyblish.api.ContextPlugin):
+    """Validate if all Background tool are set to float32 bit"""
+
+    order = pyblish.api.ValidatorOrder
+    label = "Validate Background Depth 32 bit"
+    actions = [action.RepairContextAction]
+    hosts = ["fusion"]
+    families = ["*"]
+    optional = True
+
+    comp = None
+
+    @classmethod
+    def get_invalid(cls, context):
+        cls.comp = context.data.get("currentComp")
+        assert cls.comp, "Must have Comp object"
+
+        backgrounds = cls.comp.GetToolList(False, "Background").values()
+        if not backgrounds:
+            return []
+
+        return [i for i in backgrounds if i.GetInput("Depth") != 4.0]
+
+    def process(self, context):
+        invalid = self.get_invalid(context)
+        if invalid:
+            raise RuntimeError("Found %i nodes which are not set to float32"
+                               % len(invalid))
+
+    @classmethod
+    def repair(cls):
```

Comment:
Does this actually work? I'm expecting it doesn't since it has a different signature then the repair methods on Maya validators.

I believe this should also have the "instance" as argument, which additionally means you don't need to store the `comp` on the class with `cls.comp` which, currently, might lead to unexpected resulted. (Because it's not on the class instance, but the class object.)